assets/js/components/EntityHeader.js

```diff
@@ -37,9 +38,21 @@ import { CORE_SITE } from '../googlesitekit/datastore/site/constants';
 import BackspaceIcon from '../../svg/keyboard-backspace.svg';
 import { CORE_LOCATION } from '../googlesitekit/datastore/location/constants';
 import Link from './Link';
+import classnames from 'classnames';
 const { useSelect, useDispatch } = Data;
 
-const EntityHeaderBanner = () => {
+const EntityHeader = () => {
+	const [ hasScrolled, setHasScrolled ] = useState( false );
+	useEffect( () => {
+		global.window.onscroll = function () {
+			if ( global.window.scrollY > 5 ) {
+				setHasScrolled( true );
+			} else {
+				setHasScrolled( false );
+			}
+		};
```

Comment:
We don't use the old approach of defining event listeners. If you need to add one, use the `addEventListener` method accompanied with the corresponding `removeEventListener` call in the clean-up function to avoid memory leaks and unexpected console errors.

Also, `global` already means `window` when run in the browser. So using `global.window....` doesn't make sense. Just use `global.addEventListener( ... )`.

Reply:
Thanks for the tips - will keep these in mind!